Updated TP api version and bug fixes

DamienStaebler opened the pull request:
- Fixed unable to move process to non-standard multimonitor setup [Issues]( https://github.com/KillerBOSS2019/Windows-Mover/issues/4) Thanks to @xenimax for helping me out
- removed statedUpdate keep creating new thread and call itself as It is expensive to create thread. So now it will use while loop instead which is much more efficient

DamienStaebler committed:
Updated TP api version and bug fixes

- Fixed unable to move process to non-standard multimonitor setup [Issues]( https://github.com/KillerBOSS2019/Windows-Mover/issues/4) Thanks to @xenimax for helping me out
- removed statedUpdate keep creating new thread and call itself as It is expensive to create thread. So now it will use while loop instead which is much more efficient

KillerBOSS2019 commented:
I don't want to marge it yet. because I noticed when I run the program for about 10-20minutes it suddenly uses 10% CPU. during my testing without compiling to exe. Will debug this later. (I am guessing might be the stateUpdate func causing the issue but not sure)

DamienStaebler committed:
Added Process window block list

- There are some window process like PopupHost which is for mouse hovering event.
